tools/deploy_translated_documentation.sh

```diff
@@ -26,7 +26,9 @@ RCLONE_CONFIG_PATH=$(rclone config file | tail -1)
 set -e
 
 # Clone the sources files and po files to $SOURCE_DIR/docs_source
-git clone --depth=1 $SOURCE_REPOSITORY docs_source
+git clone $SOURCE_REPOSITORY docs_source
+STABLE_COMMIT_HASH=`cat ./qiskit-commit-hash`
+git checkout $STABLE_COMMIT_HASH
```

Comment:
When we do `git checkout` here, we're still in the outer git repository.  We possibly want one of:
```suggestion
git -C docs_source checkout $STABLE_COMMIT_HASH
```
or
```suggestion
pushd docs_source
git checkout $STABLE_COMMIT_HASH
popd
```